src/Sarif.Multitool.FunctionalTests/TestData/EndColumnMustNotBeLessThanStartColumn/EndColumnLessThanStartColumnInResultLocation_Expected.sarif

```diff
@@ -0,0 +1,95 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.0.0",
+  "version": "2.0.0",
+  "runs": [
+    {
+      "tool": {
+        "name": "Sarif.Multitool",
```

Comment:
>Sarif.Multitool [](start = 17, length = 15)

hm. no indicator here that we're invoking your validate verb. this is in line with a comment someone raised in the SARIF TC (might have been me), SARIF doesn't clearly describe plug-ins/optionally invoked functionality in the format. i.e., assumes tool -> single configuration/plug-in set.

this observation probably warrants a TC tracking issue, assuming we don't already have one

Reply:
You're right. We do have the `commandLine` property of the `invocation` object (we're not populating it, but we could). 

---
In reply to: [205218073](https://github.com/Microsoft/sarif-sdk/pull/968#discussion_r205218073) [](ancestors = 205218073)

Reply:
On further reflection, I don't think the lack of "plug-ins" support is the issue here. That discussion was about the scenario where you have v1 of tool MyTool.exe, and it accepts plugins like rule sets, and you give it v2 of MySecurityRules.dll and v3 of MyAccessibilityRules.dll. Maybe you deploy them to the tool installation directory, and they're not even mentioned on the command line). Then SARIF has no place for you to mention the plug-ins or what versions they are.

Sure, a plug-in could add a whole new verb, but that's not the case here: `validate` is built-in.

Here, I think the existing `invocation.commandLine` would meet the case, but I don't think it's worth holding this merge for it.

---
In reply to: [205219537](https://github.com/Microsoft/sarif-sdk/pull/968#discussion_r205219537) [](ancestors = 205219537,205218073)

Reply:
I filed [https://github.com/oasis-tcs/sarif-spec/issues/207](https://github.com/oasis-tcs/sarif-spec/issues/207), "Support tool plug-ins."

---
In reply to: [205224952](https://github.com/Microsoft/sarif-sdk/pull/968#discussion_r205224952) [](ancestors = 205224952,205219537,205218073)